submissions/Complexity Corner 3 - Efficiency Modules/Complexity Corner 3 - Efficiency Modules.md

```diff
@@ -0,0 +1,35 @@
+
+What are efficiency modules?
+
+Like the other two modules, efficiency modules can be placed into buildings to change how they operate.  Like speed modules, efficiency modules can be placed in any module slot.  Unlike both speed and productivity modules, efficiency modules have one effect and one only, with no direct drawbacks or side-effects.  Well, one effect that has two benefits; they reduce how much power the building takes to operate, which also reduces the pollution the building produces.
+
+What results can they give?
+
+While they can reduce the strain on a power network that has been pushed to its limit, or save you resources by not having to make anywhere near as many power producers, the main benefit is reduced pollution.  A LOT of reduced pollution.  An efficiency 1 module reduces power and pollution by a staggering 30%!  An important note however is that each machine can only benefit from a max of 80% reduction, so keep that in mind when you're making use of them.
+
+Many theory-crafters like to assume clean energy for the entire factory.  That is, all the power comes from solar, which has no pollution.  The truth is that for many of us, we spend much of a playthrough relying on steam or nuclear, both of which create pollution (though nuclear does so indirectly through the fuel cell creation).  In fact, many of us never rely completely on solar.  This means that the reduced power usage also means reduced pollution from power generation.
+
+Even if you do generate all your power from solar, efficiency modules will still reduce pollution made by the machine itself, because that pollution is modified by the power usage multiplier.  In simpler words, this means that if your machine is only using 20% power because of efficiency modules, it is also producing only 20% pollution.  In this example, an Assembly Machine 2 would only produce 0.6 pollution per minute, down from the normal value of 3 per minute.
+
+To show how drastic an effect efficiency modules can have on your pollution cloud, I've prepared a factory that is roughly representative of an average player's pre-rocket factory, maybe even smaller.  It makes 45 SPM, makes all its power from solid fuel and steam, and uses mid-tier machines (steel furnaces, electric mining drills, Mk2 assemblers).  I ran this factory three times, starting each run with no pollution.  The first run was with no modules, the second run used a mix of productivity and speed 1 modules, and the third run used efficiency 1 modules everywhere.  Take a look at the pollution clouds after 1 hour.
```

Comment:
Well done one the thorough and hard work! I think you should just be concise about _why_ you started the factory from scratch each time. Essentially because you want to include pollution created by making the modules.